portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,82 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Kayla Harwell's Portfolio</h1>
+      <h2> Who am I? </h2>
+
+        <p> Hi! My name is Kayla Harwell. I am a rising junior from
+            Howard Univerity. I am 19 years old and I am from Buffalo, NY. I didn't plan on
+             majoring in CS until I was a junior in HS. Before then I had my sights set on being
+            an oncologist, but after a few medical internships I realized that wasn't for me and
+            shifted my focus towards technology,something that I'd always had a love for.</p>
+
+
+        <p> At Howard I like to spend my time involved on campus through clubs like NSBE
+            and holding E-board positions in ACM and SWE.I love Howard and its culture.
+            Everyone there is unique and I would'nt be where I am today without them. Along with this
+            oppertunity Howard has opened many doors for me to grow and learn.</p>
+      
+
+      <h2> What Projects Have I Worked On? </h2>
+        <ul>
+            <li> "Absent Future" - This project is my most recent. It is a first person shooter game
```

Comment:
nit/idea: is there anything online you could link to? e.g. articles written about it or a website you made during the hackathon etc.